apis/r/R/Factory.R

```diff
@@ -49,6 +49,7 @@ SOMADataFrameCreate <- function(
       schema,
       index_column_names = index_column_names,
       platform_config = platform_config,
+      timestamps = rep(tiledb_timestamp, 2),
```

Comment:
Marking once for the entire file: `$create()` should not accept timestamps; instead, it should be using the timestamps cached in the object during `$new()`

Reply:
I just tried a baby-sized version of that change, and given that the tests _currently_ do this via `$create()` I have shrapnel.

Reply:
Addressed in #2930